config/rbac/role.yaml

```diff
@@ -44,22 +45,6 @@ rules:
   verbs:
   - list
   - watch
-- apiGroups:
-  - '*'
-  resources:
-  - '*'
-  verbs:
-  - get
```

Comment:
To maintainers: Please note that this overly permissive rule "allow to _get_ any resource" was removed without any replacement. 

Should I keep it untouched, or is it safe to delete it?

Reply:
In helm, we remove it based on configs, but it's required using raw manifests

Reply:
Thank you, @JorTurFer! I reverted the deletion of that particular rule.